src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -475,7 +475,7 @@ private void OnPreviewControlStateChanged(object sender, EventArgs e)
                     }
                 case PreviewControl.State.Expanded:
                     {
-                        if (!IsMouseOver && !preview.IsMouseOver)
+                        if (!IsMouseOver && !preview.IsMouseOver && !preview.StaysOpen)
```

Comment:
MAGN-9364 Pinned preview bubble does not automatically show updated detailed view when ports are disconnected/reconnected

After Preview was expanded, it should be collapsed in case, when: mouse is not over node and mouse is not over preview itself and preview is not pinned. Just in that case we move it in Condensed state.